package.json

```diff
@@ -26,7 +26,7 @@
     "test-unit": "npm run build && jest test/unit/",
     "test-integration": "npm run build && jest test/integration",
     "test-travis": "jest --runInBand --testNamePattern='^((?!@slow).)*$' test/",
-    "test-unit-travis": "jest --runInBand test/unit/",
+    "test-unit-travis": "jest --runInBand test/unit/cloudantBaseService.test.js",
```

Comment:
Local "optimisation" included in error? Otherwise what is the moitvation for reducing the set of unit tests?

Reply:
0a7fdd1